### ISSUE TITLE
calenders showing up stacked vertically instead of horizontally

### ISSUE THREAD

**Issue by nugururaparla** [skratchdot/react-bootstrap-daterangepicker#173]
When using the DateRangePicker, the calendars are showing up stacked vertically by default. Setting the display property to 'inline-flex' instead of 'block' would fix the orientation but I am having trouble figuring out where to change. I tried using  containerStyles={{ display: "inline-flex !important " }}

<DateRangePicker startDate={this.state.fromDate} endDate={this.state.toDate} 
                               onApply={this.applyDateChange.bind(this)} maxDate={moment()} 
                               showDropdowns={true} containerStyles={{ display: "inline-flex !important" }} 
                               ranges={this.state.ranges}>
             <button>Date Range</button>
</DateRangePicker>

I also tried changing the css for .daterangepicker but did not work as expected. (I changed the 'diplay: none' for .daterangepicker to 'display: inline-flex')

what is the correct method for controlling/updating the styles?

![image](https://user-images.githubusercontent.com/15654859/40253987-da7b2fd6-5aaf-11e8-844d-187119c83416.png)

**Comment by skratchdot** [comment]
I think you should remove the `!important`, and just use:

```
<DateRangePicker containerStyles={{ display: "inline-flex" }} />
```

Also, make sure you are using v4+

**Comment by nugururaparla** [comment]
I updated the bootstrap to use the latest stable version and tried without the !important flag. Didn't work right away.  Then I compared the daterangepicker.css that I am using with the one in dangrossman/daterangepicker/daterangepicker.css github project and I noticed a few differences. The css file I was using had .calendar instead of .drp-calendar. Once I removed all the differences I started seeing the calendars show up correctly.

Thanks for the prompt reply.
